frontend/src/wailsjs/runtime/runtime.d.ts

```diff
@@ -8,6 +8,14 @@ export interface Size {
     h: number;
 }
 
+export interface RGBA {
```

Comment:
This looks like it has been generated by a previous version of wails. Could you ensure you generate this using version 32 beta ?

Reply:
https://github.com/vegaprotocol/vegawallet-desktop/issues/189 I have added this ticket to track this